hash/msx1_cart.xml

```diff
@@ -12580,7 +12472,6 @@ kept for now until finding out what those bytes affect...
 		</part>
 	</software>
 
-<!-- Bee Card (possibly also ROM) -->
 	<software name="yakyukyo">
 		<description>Yakyu Kyo (Jpn)</description>
```

Comment:
Did you want to fully remove this one from msx1_cart.xml?

Reply:
The game was released on Bee Card and regular cartridge; see also: https://www.generation-msx.nl/software/hudson-soft/baseball-craze/release/671/

Reply:
Is it worth leaving the note that this might not exist on cartridge? I couldn't find any evidence for a cart version after doing a bit of google, auction, image search etc. I tried various combos of these keywords: MSX ハドソン 野球狂 ROM カートリッジ.

Reply:
Yes, I agree. The section on that generation msx page only mentions it but also has no pictures of an actual cartridge.